src/org/labkey/test/tests/GpatAssayTest.java

```diff
@@ -361,82 +310,6 @@ public void testRenameAssayDesign()
 
     }
 
-    @Test
-    public void testRenameSharedAssayDesign()
```

Comment:
I would have expected this to be refactored to match the React UI instead of deleting the test case completely. Is it redundant or irrelevant now?

Reply:
The other test cases from this file that "take a file from the file system, click Import Data > Create Standard Assay Design, and import the file" were refactored here to use the React UI. This test case does that but was selecting for the assay design to be created in the /Shared location. That part is no longer supported in the new UI/UX (as noted in the PR comment) so I just removed this test case.

Reply:
Got it - I missed that in the other PR's description.